create_driver/src/create_driver.cpp

```diff
@@ -31,22 +31,31 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #include <string>
 
-CreateDriver::CreateDriver(ros::NodeHandle& nh)
+
+CreateDriver::CreateDriver(std::shared_ptr<rclcpp::Node> nh)
   : model_(create::RobotModel::CREATE_2),
     nh_(nh),
-    priv_nh_("~"),
-    diagnostics_(),
+    priv_nh_(rclcpp::Node::make_shared("_")),
```

Comment:
Since parameters are "private" to a node in ROS 2, I don't think we need to a separate node anymore. Please remove `priv_nh_` and just use `nh_` for declaring parameters below.

Reply:
Done.